vcpkg.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "merry-mem",
+  "version-string": "0.1.0",
+  "features": {
+    "test": {
+      "description": "Libraries required for testing.",
+      "dependencies": [
+        "catch2"
+      ]
+    }
+  }
+}
```

Comment:
What is this file for? I know you can't comment in a `*.json` file. I'm just curious.

Reply:
This is what vcpkg calls the "manifest". It's basically like package.json. You can define dependencies and group them into features. Its presence in the project root lets vcpkg know to work in "manifest mode".

From: https://learn.microsoft.com/en-us/vcpkg/users/manifests